static-site/content/blog/2025-03-31-annual-update-march-2025.md

```diff
@@ -0,0 +1,74 @@
+
+In planning work for the March 2025 to March 2026 period, we've emphasized infrastructural priorities that position us in the best way to respond to emerging outbreaks, epidemics and pandemics. This includes:
+
+1. **Support for large datasets via streamtrees and other improvements to Auspice**. As noted above, a first version of streamtrees is almost live. However, this will need further refinement in Auspice and will need work in Augur to support delineation of evolutionarily or epidemiologically relevant clades. Future pandemics or impactful epidemics will have a deluge of data and supporting trees with more than \~4000 tips is a key priority. With streamtrees in place, existing Augur workflows should be able to handle 20k-30k datasets without too much difficulty.
+
+2. **Support for frequencies analyses and visualization**. We've continued with live SARS-CoV-2 frequencies fitness analysis and visualization at [/sars-cov-2/forecasts](/sars-cov-2/forecasts) and separately with influenza frequencies visualization at [github.com/nextstrain/flu\_frequencies](https://github.com/nextstrain/flu_frequencies). Frequencies analysis (as opposed to phylogenetic analysis) allows nearly arbitrary scaling to dataset size and greatly facilitates analysis of lineage and mutation fitness. We believe that a system of automated nomenclature \+ streamtrees view in Auspice \+ frequencies view would provide a foundation for large-scale evolutionary analysis. We didn't make much progress on this Aim over the previous year, but we hope to establish a clear direction here in 2025\.  
+
+3. **Continued build out of automated workflows to 20-30 pathogens**. We believe this pattern of automated ingest via NCBI and automated phylogenetics to be a strong basis and would like to continue to establish a foundation of real-time analysis across pathogens. Currently we have 15 automated pathogens after adding an additional 9 over the course of the previous year. Current contenders for automation include CCHV, hepatitis B virus, Ebola virus, EV-D68 and tuberculosis (TB), which would bring us to 20. Work is actively proceeding on TB, which would represent the first automated bacterial workflow, though a few non-automated examples currently exist (including [Y. pestis](/community/ktmeaton/yersinia-pestis/maximum-likelihood/all?m=num_date) and [cholera](/groups/blab/cholera/7PET?c=subregion)).
+
+4. **Improvements to Nextclade.** Our primary plans for the coming year are to (i) generate annotations for query sequences that can be used to submit these sequences to NCBI, (ii) Allow users to analyze collections of sequences that map to difference reference sequences, e.g. different segments of influenza viruses, (ii) support for segmented viruses and multi-segment nomenclatures. The latter is a bigger project that we still need to scope in greater detail, but we envision such functionality to help analysis in scenarios like the recent H5N1 outbreaks.
```

Comment:
<nit> For consistency with the other points:

```suggestion
4. **Improvements to Nextclade**. Our primary plans for the coming year are to (i) generate annotations for query sequences that can be used to submit these sequences to NCBI, (ii) Allow users to analyze collections of sequences that map to difference reference sequences, e.g. different segments of influenza viruses, (ii) support for segmented viruses and multi-segment nomenclatures. The latter is a bigger project that we still need to scope in greater detail, but we envision such functionality to help analysis in scenarios like the recent H5N1 outbreaks.
```

Reply:
Resolved in 3576ccbf888df9289f183b379e003f85d1a389fe.